packages/docs/components/tab.md

```diff
@@ -4,44 +4,49 @@ Tabs are a navigational component used to organize content by grouping similar i
 
 <figure class="nimatron--example">
   <div class="nimatron--rendered">
-    <OdsTabs label="Mocha Joe's Menu" :active="tabs.active" :tablist="tabs.tablist" :id="tabs.id">
-      <template slot="donuts-tab">
-        <p>Tab Panel: Donuts</p>
-      </template>
-      <template slot="pastries-tab">
-        <p>Tab Panel: Pastries</p>
-      </template>
-      <template slot="coffee-tab">
-        <p>Tab Panel: Coffee</p>
-      </template>
+    <OdsTabs label="User profile options" :active="tabs.active" :tablist="tabs.tablist" :id="tabs.id">
+      <template slot="tab-applications"><p>Tabpanel: Applications</p></template>
+      <template slot="tab-groups"><p>Tabpanel: Groups</p></template>
+      <template slot="tab-profile"><p>Tabpanel: Profile</p></template>
+      <template slot="tab-devices"><p>Tabpanel: Devices</p></template>
+      <template slot="tab-omm"><p>Tabpanel: OMM</p></template>
     </OdsTabs>
   </div>
 
 ```html
-<div id="example-0" class="ods-tabs">
-  <div role="tablist" aria-label="Mocha Joe's Menu" class="ods-tabs--tablist" style="
-    --ods-tabs-indicator-width:99px; 
-    --ods-tabs-indicator-pos-x:0px;
-  ">
-    <button role="tab" id="donuts-tab" tabindex="0" aria-controls="donuts-tab-tabpanel" class="ods-tabs--tab" aria-selected="true">
-        Donuts
+<div id="user-profile-tabs" class="ods-tabs" label="User profile options">
+  <div role="tablist" aria-label="" class="ods-tabs--tablist">
+    <button role="tab" id="tab-applications" aria-selected="true" tabindex="0" aria-controls="tab-applications-tabpanel" class="ods-tabs--tab">
```

Comment:
As an FYI - this isn't a [required aria attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role) and isn't compatible with [web components](https://github.com/w3c/aria/issues/864). [Material UI](https://developers.google.com/web/fundamentals/web-components/shadowdom) and [Polaris](https://github.com/Shopify/polaris-react/blob/2b816ef17a863d39d988fe95b631e92fdd1e19f8/src/components/Tabs/Tabs.tsx) do not use these labels either.

In the future it would be nice to outline attributes as either **MUST** or *SHOULD* be implemented, since it'll depend on the technology used by consumers.